scripts/search.py

```diff
@@ -131,23 +131,22 @@ def filter_forward_pairs_by_reverse_pairs(forward_blast_high_scoring_pairs, reve
     filterable_forward_blast_results = list(forward_blast_high_scoring_pairs)
     print(">> Checking if forward hit subjects have better reciprocal hits than query.")
     for hit in forward_blast_high_scoring_pairs:
-        query_protein = blast_graph.getVertex(hit[0])
-        subject_protein = blast_graph.getVertex(hit[1])
+        subject_protein = blast_graph[hit[1]]
 
-        top_back_hit_score = 0
         # Find the top score of the best reciprocal BLAST hit
-        for backHit in subject_protein.getConnections():
-            back_hit_score = subject_protein.getWeight(
-                backHit)  # The edge weight between the subject and its reciprocal BLAST hit is the BLAST score
+        top_back_hit_score = 0
+        for back_hit_id in subject_protein:
+            back_hit_score = get_edge_attribute(subject_protein, back_hit_id,
+                                                'bitscore-rev', none_value=-1)
             if back_hit_score >= top_back_hit_score:
                 top_back_hit_score = back_hit_score
 
         # Check if the query is the best reciprocal BLAST hit for the subject
         delete_hit = False
-        if query_protein in subject_protein.getConnections():
+        if hit[0] in subject_protein:
```

Comment:
@jmtsuji To improve readability, `hit[o]` should be assigned to a variable like before. 
The old code had " query_protein = blast_graph.getVertex(hit[0])" so now it should be "query_protein = blast_graph[hit[0]]" and you should use query_protein varaible rather than hit[0] in the follow on code.

Reply:
@LeeBergstrand I've now made the variables clearer -- let me know what you think. Thanks!